### PR TITLE
Added app name

### PR DESCRIPTION
**Pull request by Antony1060**
This just adds the ability for a bit more customization.

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -247,6 +247,8 @@ type Configuration struct {
 	// if the debug flag is passed through the command line arguments.
 	Debug bool
 
+	AppName string `json:"app_name" yaml:"app_name"`
```
Comment (schrej) [inline config/config.go]:
```suggestion
	AppName string `default:"Pterodactyl" json:"app_name" yaml:"app_name"`
```

**Review by schrej — CHANGES_REQUESTED** [review]

**Review by schrej — APPROVED** [review]

**Comment by ririko5834** [comment]
When this will be added?

**Comment by FoksVHox** [comment]
> When this will be added?

It doesn't help to comment "When will this be added", "any update". Only comment if you've anything relating to the PR. 

@Antony1060 What's up with the failing test?

**Comment by Antony1060** [comment]
> What's up with the failing test?

That test seems to fail on all of the other PRs. I don't think I can do anything about that.

**Comment by hwalker928** [comment]
Shouldn't the default be "Pterodactyl Daemon", so you have the option to set it to whatever you want, regardless if it ends in Daemon or not?

**Comment by FoksVHox** [comment]
> Shouldn't the default be "Pterodactyl Daemon", so you have the option to set it to whatever you want, regardless if it ends in Daemon or not?

I wouldnt say so as Wings is a daemon

**Pull request merged**